File: Programmers/Lv.2/JadenCase 문자열 만들기/12951.js
function solution(s) {
    let answer = '';
    
    let firstFlag = true;
    s.split('').forEach(v => {
       if (v === ' ') {
           answer += v;
       } else if (!isNaN(v)) {
           answer += v;
       } else if (firstFlag) {
           answer += v.toUpperCase();
       } else {
           answer += v.toLowerCase();
       }
        
        firstFlag = (v === ' ');
    });
    
    return answer;
}

File: Programmers/Lv.2/JadenCase 문자열 만들기/12951.java
class Solution {
    public String solution(String s) {
        String answer = "";
        
        boolean firstFlag = true;
        for (char c : s.toCharArray()) {
            if (c == ' ') {
                answer += c;
            } else if (Character.isDigit(c)) {
                answer += c;
            } else if (firstFlag) {
                answer += (c + "").toUpperCase();
            } else {
                answer += (c + "").toLowerCase();
            }
            
            firstFlag = (c == ' ');
        }
        
        return answer;
    }
}
